lms/envs/common.py

```diff
@@ -490,6 +491,7 @@
     'REFRESH_TOKEN_EXPIRE_SECONDS': 7776000,
     'SCOPES_BACKEND_CLASS': 'openedx.core.djangoapps.oauth_dispatch.scopes.ApplicationModelScopes',
     'SCOPES': dict(OAUTH2_DEFAULT_SCOPES, **{
+        'user_id': _('Know your user_id'),
```

Comment:
I believe this message gets displayed to users. Maybe something like this instead?
```
'user_id': _('Retrieve your user identifier'),
```

Reply:
Done.  Thanks.  I had copied the less-than-ideal messages from above for email and profile.